src/Microsoft.VisualStudio.ProjectSystem.Managed/Tree/Dependencies/DependenciesTreeViewProvider.cs

```diff
@@ -320,16 +338,14 @@ private static IProjectTree CleanupOldNodes(IProjectTree rootNode, IEnumerable<I
             ProjectTreeFlags? additionalFlags = null,
             ProjectTreeFlags? excludedFlags = null)
         {
-            IRule? rule = null;
-            if (dependency.Flags.Contains(DependencyTreeFlags.SupportsRuleProperties))
-            {
-                rule = await _treeServices.GetBrowseObjectRuleAsync(dependency, targetedSnapshot.Catalogs);
-            }
+            IRule? browseObjectProperties = dependency.Flags.Contains(DependencyTreeFlags.SupportsRuleProperties)
```

Comment:
Technically this is not "properties", but a description of the properties.

Reply:
Agree. I flowed the name back from arguments in `IDependenciesTreeServices`. I'll update all instances to `browseObjectRule`.

Reply:
Actually the name comes from CPS (`IProjectTree`).

Reply:
CPS uses `browseObjectProperties` in several places. Think I'll leave it alone rather than fight it. If you disagree we can change it in a follow up PR.